workers.py

```diff
@@ -79,59 +80,57 @@ def _inference(model: dict, data: pd.DataFrame) -> dict:
     return inf.predict()
 
 
-def prediction_worker(job: dict, next_service: str) -> Thread:
+def prediction_worker(id: str, raw_data: bytes, next_service: str) -> Thread:
     def worker() -> None:
         thread = current_thread()
         logger.debug('%s: Worker started', thread.name)
 
-        try:
-            batch_id, batch_data = job['id'], job['data']
-        except KeyError:
+        if not all((id, raw_data)):
             logger.error("%s: Invalid Job data, terminated.", thread.name)
             return
 
-        logger.info('%s: Job ID %s: Started...', thread.name, batch_id)
+        logger.info('%s: Job ID %s: Started...', thread.name, id)
         try:
-            data = pd.DataFrame.from_dict(batch_data)
+            data = pd.read_csv(BytesIO(raw_data))
```

Comment:
@TreeinRandomForest is there some important difference in `from_dict` and `read_csv` from pandas? I don't see a problem in that change, as long `data` will be the same dataframe....